src/components/form-grid/form-grid.stories.js

```diff
@@ -0,0 +1,109 @@
+import { html } from "lit-html";
+import "../form";
+import "../form-grid";
+import "../input-field";
+import "../button";
+import "../form-message";
+import styles from "./styles.scss";
+import formStyling from "../form/styles.scss";
+import inputFieldStyling from "../input-field/styles.scss";
+import buttonStyling from "../button/styles.scss";
+import formMessageStyle from "../form-message/styles.scss";
+import { stylesheet, docsIntro } from "../../../.storybook/utils.js";
+import { args, argTypes } from "../grid/config";
+
+export default {
+  title: "native-elements/vl-form-grid",
+  decorators: [
+    (story) =>
+      html`${stylesheet(
+        `${styles}${formStyling}${inputFieldStyling}${buttonStyling}${formMessageStyle}`
+      )}${story()}`,
+  ],
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          stylesheets: ["form-grid"],
+          root: "form-grid",
+          intro: "Class that enables a grid layout in a form.",
+        }),
+      },
+    },
+  },
+  args: { ...args },
+  argTypes: { ...argTypes },
+};
+
+export const Default = ({
+  stacked,
+  stackedSmall,
+  stackedLarge,
+  alignStart,
+  alignCenter,
+  alignEnd,
+  alignSpaceBetween,
+  alignSpaceAround,
+  vTop,
+  vCenter,
+  vBottom,
+  vStretch,
+}) => html`
+  <form>
```

Comment:
Ik zou dezelfde wrapper width gebruiken als bij de vl-form, om wat consistent te blijven.

Reply:
Is de form die je gebruikt niet best een vl-form? Bij de form-column doe je dat wel.

Reply:
Fixed.